### PR TITLE
fix(aio): Fix deploy to firebase script: should use < for string oper…

### PR DESCRIPTION
**Pull request by tinayuangao**
…ations

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Fix aio script error: https://travis-ci.org/angular/angular/jobs/264850103.

Current script uses the value of `$majorVersion` and `$majorVersionStable` as variable names and throws variable unbound error.

For string operations we should use `>` or `<` instead of `-ge` operators.

```
[ x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Script use the value of `$majorVersion` and `$majorVersionStable` as string, and compare their values

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by mhevery — APPROVED** [review]

File: aio/scripts/deploy-to-firebase.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ else
   readonly majorVersionStable=${STABLE_BRANCH%%.*}
 
   # Do not deploy if the major version is not less than the stable branch major version
-  if [[ $majorVersion -ge $majorVersionStable ]]; then
+  if [[ !( "$majorVersion" < "$majorVersionStable" ) ]]; then
```
Comment (gkalpak) [inline aio/scripts/deploy-to-firebase.sh]: OOC, why this change? What is wrong with using `-ge`?
